benchmarks/IO.py

```diff
@@ -28,16 +40,20 @@ def time_ak_write(N_per_locale, numfiles, trials, dtype, path, seed, parquet, co
         a = ak.randint(0, 1, N, dtype=ak.float64, seed=seed)
     elif dtype == "uint64":
         a = ak.randint(0, 2**32, N, dtype=ak.uint64, seed=seed)
+    elif dtype == "str":
+        a = ak.random_strings_uniform(1, 16, N, seed=seed)
 
     writetimes = []
     for i in range(trials):
-        for j in range(numfiles):
-            start = time.time()
-            a.to_hdf(f"{path}{j:04}") if not parquet else a.to_parquet(
-                f"{path}{j:04}", compression="snappy" if compressed else None
-            )
-            end = time.time()
-            writetimes.append(end - start)
+        for comp in comps:
+            if comp in COMPRESSIONS:
+                for j in range(numfiles):
+                    start = time.time()
+                    a.to_hdf(f"{path}{j:04}") if not parquet else a.to_parquet(
+                        f"{path}{comp}{j:04}", compression=None if comp == "none" else comp
+                    )
+                    end = time.time()
+                    writetimes.append(end - start)
```

Comment:
It seems like we're averaging the times of all the compressions together? Would it make more sense to have separate timings for each compression type?

Reply:
Added averaging for each compression type